quickwit-cli/src/cli.yaml

```diff
@@ -379,6 +379,35 @@ subcommands:
         settings:
             - ArgRequiredElseHelp
         subcommands:
+            - add:
+                about: Adds a new source.
+                args:
+                    - index:
+                        about: ID of the target index.
+                        long: index
+                        value_name: INDEX
+                        required: true
+                    - id:
```

Comment:
there's a little inconsistency with the similar command `quickwit index create --index`.

Using `--id` seems relevant also for `index create`. Concerning create/add verb, I'm hesitating to  keep both.
- `quickwit index create --id <INDEX ID>`.
- `quickwit source create --id <SOURCE ID>`.

or 

- `quickwit index add --id <INDEX ID>`.
- `quickwit source add --id <SOURCE ID>`.

Reply:
I think `add` makes sense for a source because you're really adding/appending the source to an index. When creating a new index, you're not really adding the index to anything. Although you could argue, you add the index to the metastore :)

If we're willing to sacrifice some semantic accuracy for more consistency, I'd go with `create`.

Reply:
As far as command parameter names, yeah `index create --id` seems better. I can make that change f you want.

Reply:
> I think add makes sense for a source because you're really adding/appending the source to an index. When creating a new index, you're not really adding the index to anything. Although you could argue, you add the index to the metastore :)

Yeah, I made the same rationale. Let's keep `add` for this minor command.


> As far as command parameter names, yeah index create --id seems better. I can make that change f you want.

Not sure if it's worth it :/. Let's do that later in another PR.